nordeus/crowd-user-bundle/2.0/config/packages/nordeus_crowd_user.yaml

```diff
@@ -0,0 +1,16 @@
+nordeus_crowd_user:
+    # REQUIRED:
+    crowd_application_name: '%env(NORDEUS_CROWD_APPLICATION_NAME)%'
+    crowd_service_url: http://crowd.your_domain.com:8095
+    sso_cookie_domain: your_domain.com
+    crowd_application_password: '%env(NORDEUS_CROWD_APPLICATION_PASSWORD)%'
+    roles_to_groups:
+        ROLE_USER: [ users ]
+        ROLE_ADMIN: [ admins, superadmins ]
+    # OPTIONAL:
```

Comment:
There is no need of showing these optional parameters and their default values. Keep the ones above (because they are great) and remove the ones below.

Reply:
Ok